envoy/common/execution_context.h

```diff
@@ -30,6 +60,12 @@ class ExecutionContext : public StreamInfo::FilterState::Object, NonCopyable {
   // protected because it should only be called by ScopedExecutionContext.
   virtual void deactivate() PURE;
 
+private:
+  static std::atomic<bool>& enabled() {
+    static std::atomic<bool> value = false;
+    return value;
+  }
```

Comment:
If this necessary? Now, compile flag is used to control the feature will be compiled or not. Or you still want a runtime feature flag in your implementation?

Anyway, `MUTABLE_CONSTRUCT_ON_FIRST_USE` is perferred rather than define the static by yourself.

Reply:
Changed to `MUTABLE_CONSTRUCT_ON_FIRST_USE`.